versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java

```diff
@@ -537,7 +537,8 @@ public static Stream<List<String>> progressivelyManyKeyNames() {
         .mapToObj(
             i ->
                 IntStream.range(1, i)
-                    .mapToObj(j -> Long.toString(Math.abs(random.nextLong()), Character.MAX_RADIX))
+                    .mapToObj(
+                        j -> Long.toString(random.nextLong() & Long.MAX_VALUE, Character.MAX_RADIX))
```

Comment:
Ideally, we need to choose the seed (line 533) again in this case. The changed names are not guaranteed to expose the bug, whose handling that the test is meant to validate.

Reply:
sorry i missed reading the context of this change (that we were testing an edge condition with explicitly crafted test data)

Reply:
Yea - let's have a separate issue for that.

Reply:
well, to be fair, the comment was hidden outside of the GH "changed lines" window :)

Reply:
I ran a quick test with the new name generator and old code base: compared to the old generator the number of failed test cases decreased from 40 to 33 (out of 200). I think the test data is still good enough. I do not think we need to change the seed.